nservicebus/extending/queuecreation.md

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
best i can think of is a page under transports http://docs.particular.net/nservicebus/transports/  ??